develop.md

```diff
@@ -1,8 +1,22 @@
 # Development Guide
 
-
 ## [Javadocs](https://gaucho-matrero.github.io/altoclef/)
 
+## Formatting
```

Comment:
This feels like a general tutorial for formatting, while good advice I don't see how it's relevant to altoclef developers themselves.